scanner/matcher/matcher.go

```diff
@@ -21,35 +21,45 @@ import (
 	"github.com/stackrox/rox/scanner/datastore/postgres"
 	"github.com/stackrox/rox/scanner/enricher/fixedby"
 	"github.com/stackrox/rox/scanner/enricher/nvd"
+	"github.com/stackrox/rox/scanner/enricher/partial"
 	"github.com/stackrox/rox/scanner/internal/httputil"
 	"github.com/stackrox/rox/scanner/matcher/updater/distribution"
 	"github.com/stackrox/rox/scanner/matcher/updater/vuln"
 )
 
-// matcherNames specifies the ClairCore matchers to use.
-var matcherNames = []string{
-	"alpine-matcher",
-	"aws-matcher",
-	"debian-matcher",
-	"gobin",
-	"java-maven",
-	"oracle",
-	"photon",
-	"python",
-	"rhel-container-matcher",
-	"rhel",
-	"ruby-gem",
-	"suse",
-	"ubuntu-matcher",
-}
+var (
+	// matcherNames specifies the ClairCore matchers to use.
+	matcherNames = []string{
+		"alpine-matcher",
+		"aws-matcher",
+		"debian-matcher",
+		"gobin",
+		"java-maven",
+		"oracle",
+		"photon",
+		"python",
+		"rhel-container-matcher",
+		"rhel",
+		"ruby-gem",
+		"suse",
+		"ubuntu-matcher",
+	}
+
+	enrichers = []driver.Enricher{
```

Comment:
The new `enrichers` variable appears to be unused, looks like it should be used here:

https://github.com/stackrox/stackrox/blob/b71c875e74806d0171892abe4140c8584412d3f5/scanner/matcher/matcher.go#L134-L137

Reply:
oops